### PR TITLE
[MTB] added readme

### PR DESCRIPTION
**Pull request by phoenixking25**
/assign @JimBugwadia 
Added basic info and commands to build the project.

### CODE REVIEW

File: benchmarks/kubectl-mtb/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+This is a kubectl plugin of the multi-tenancy benchmarks, to validate the the multi-tenancy in the K8s cluster.
+
+## Building the plugin
+
+Makefile is used to automate the things at the local level.
+**Prerequisites** : Make sure you have working GO environment in your system.
+
+### Make commands
+
+- **make kubectl** : To build the project and copy the binary file to the PATH.
```
Comment (JimBugwadia) [inline benchmarks/kubectl-mtb/README.md:12]: would suggest we call this "make mtb" or "make kubectl-mtb" to avoid confusion with kubectl.

**Comment by JimBugwadia** [comment]
Same comment on these folders:

https://github.com/kubernetes-sigs/multi-tenancy/tree/master/benchmarks/kubectl-mtb/cmd/kubectl

https://github.com/kubernetes-sigs/multi-tenancy/tree/master/benchmarks/kubectl-mtb/internal/kubectl

**Comment by rjbez17** [comment]
/hold 

until comments addressed

**Pull request merged**